index.html

```diff
@@ -80,6 +80,216 @@ <h2 class="purple">LET'S CONNECT</h2> <br>
+ 
+                 <ul class="tech-stack">
+                     <li>HTML</li>
+                     <li>CSS</li>
+                     <li>JavaScript</li>
+                 </ul>
+ 
+                 <button class="btn" type="button">See Project</button>
+             </article>
+
+             <article class="card">
+                <img src="images/project3.PNG" alt="project1">
+                <h2>Tonic</h2>
+                <ul>
+                 <li class="no-bullet">CANOPY</li>
+                 <li class="project-tag"> 
+                     <span> <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                     </svg>
+                      Back End Dev
+                     </span>
+                 </li>
+                 <li class="project-tag">
+                      <span> <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                     </svg>
+                      2015
+                     </span>
+                 </li>
+                </ul>
+                <p>A daily selection of
+                  privately personalized reads; no accounts or sign-ups required.</p>
+ 
+                 <ul class="tech-stack">
+                     <li>HTML</li>
+                     <li>CSS</li>
+                     <li>JavaScript</li>
+                 </ul>
+ 
+                 <button class="btn" type="button">See Project</button>
+             </article>
+
+             <article class="card">
+                <img src="images/project2.PNG" alt="project1">
+                <h2>Multi Post Stories</h2>
+                <ul>
+                 <li class="no-bullet">CANOPY</li>
+                 <li class="project-tag"> 
+                     <span> <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                     </svg>
+                      Back End Dev
+                     </span>
+                 </li>
+                 <li class="project-tag">
+                      <span> <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                     </svg>
+                      2015
+                     </span>
+                 </li>
+                </ul>
+                <p>A daily selection of
+                  privately personalized reads; no accounts or sign-ups required.</p>
+ 
+                 <ul class="tech-stack">
+                     <li>HTML</li>
+                     <li>CSS</li>
+                     <li>JavaScript</li>
+                 </ul>
+ 
+                 <button class="btn" type="button">See Project</button>
+             </article>
```

Comment:
- [ ] As stated in the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). but for For each project the image are broken as shown below in screenshot , the path you use for  image `<img src="images/microverse_works.PNG" alt="project1">` has no image for microverse_works.PNG, project4.PNG, project3.PNG and project2.PNG in your image folder, Kindly Fix it.
- Your Image Folder 
![Capture](https://user-images.githubusercontent.com/93598189/235237860-75bbe49b-4857-4196-9fba-2009c1c9c82e.PNG)

- Your Project Card design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/235238082-e4bdd79e-b5d0-412a-9866-d9f2ff6f71fc.PNG)

- The Figma Project design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/235238407-1e95ce02-5b03-4f9c-8679-c1618565aba6.PNG)